baaahs/sparklemotion
# Adds mongodb

Steps to test this out and get it to work:

1. install docker `curl -fsSL get.docker.com -o get-docker.sh && sh get-docker.sh`
2. run `./gradlew initMongo` (Only need to do this once, ever)
3. run `./gradlew startMongo`

You can verify mongodb is running by running `docker ps`, which should look something like this:
```
🌈  sparklemotion  🚀 (っ◔◡◔)っ[̲̅ docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                  NAMES
de72599f2e8c        mongo:4.0.10        "docker-entrypoint.s…"   6 seconds ago       Up 4 seconds        0.0.0.0:27017-27019->27017-27019/tcp   mongodb
```



Work left to do: I'm totally not sure if this is written right, first time really looking into Kotlin, so I'm sure some stuff needs to change? I'm also not sure how to call any of the methods from my `MongoDB` class from within a gadget or something, when a gadget updates, or if we send a websocket message to save a new EyePreset setting?
